### ISSUE TITLE
Changing nodetype doesn't correct defaultValues

### ISSUE THREAD

**Issue by neos-bot** [neos/neos-development-collection#1084]
Jira issue originally created by user @hhoechtl:

You have the nodeType ThreeColumn, if you change to TwoColumn the NodeConverter sets all DefaultValues according to the new nodeType.
But as the property "layout" is already set it's not changed and therefore contains an invalid value which initially destroys the rendering. Only after the new rendered node is given to the frontend the inspector states, that the layout value has been changed and you need to "Apply" this change a second time.

So it would be necessary to change the properties that exist in both nodeTypes to the "allowed" values of the new nodeType. But as for layout this is only a JS selection list and not a real enum that could be checked this is currently not possible.
Simply overwriting the properties with the new default values would probably overwrite changes for e.g. a title field that is not supposed to. Therefore there must be either a flag for the properties that should be overwritten when changing nodeType or comparable enums of valid values.

 Jira-URL: https://jira.neos.io/browse/NEOS-1393

**Comment by kdambekalns** [comment]
This issue is very old by now and has not been updated for a while.

Chances are, it is solved or otherwise no longer relevant, so I close it.

Feel free to reopen, if this is still an issue and you can provide new information.
